src/commands/installDocker.ts

```diff
@@ -0,0 +1,56 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE.md in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { IActionContext } from 'vscode-azureextensionui';
+import { ext } from '../extensionVariables';
+import { localize } from '../localize';
+import LocalOSProvider from '../utils/LocalOSProvider';
+import { DockerInstallerBase } from './dockerInstallerBase';
+
+class WindowsDockerInstaller extends DockerInstallerBase {
```

Comment:
IMO these three classes make more sense to be in the same file as `DockerInstallerBase`, or otherwise files of their own, rather than in here.

Reply:
I prefer to keep the base class separate. These 3 classes don't have much logic other than providing the data. I think having it here will give you full picture. Having a separate file each one will be an overkill.

Reply:
In that case they should move to DockerInstallerBase.

Reply:
I agree, they should either be in separate files or the base class file.